sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,252 @@
+
+The following table summarizes the main design differences between these systems:
+
+| Feature                | WBTC            | tBTC             | rBTC             | sBTC (this SIP)    |
+|------------------------|-----------------|------------------|------------------|--------------------|
+| Spending threshold      | 2 of 3          | 51 of 100        | 5 of 9           | 11 of 15           |
+| Bitcoin finality        | No              | No               | Yes              | Yes                |
+| Expected Peg-in speed   | 1 hour          | 1-3 hours        | 16 hours         | 0.5 hours          |
+| Expected Peg-out speed  | 1 hour          | 3-5 hours        | 33.3 hours       | 1 hour             |
+| Custodian rotation      | No              | Yes              | No               | Yes                |
+| Fee structure           | % of BTC moved  | % of BTC moved   | Transaction fees | Transaction fees   |
+
+
+In conclusion, the sBTC system shares similarities with existing models but introduces some key distinctions:
+- **Bitcoin Finality:** sBTC inhereits Bitcoin finality from [Stacks 3.0](https://github.com/stacksgov/sips/blob/feat/sip-021-nakamoto/sips/sip-021/sip-021-nakamoto.md), which ensures that sBTC transactions receive the same level of security provided by the Bitcoin network.
+- **Faster Deposit & Withdrawal Times:** sBTC enables BTC withdrawals without the long delays associated with block confirmations in other systems. This is achieved through the finality rules described in [Stacks 3.0](https://github.com/stacksgov/sips/blob/feat/sip-021-nakamoto/sips/sip-021/sip-021-nakamoto.md).
+
+
+## Activation
+
+sBTC is designed to activate on Stacks 3.0 as defined in [SIP-021](https://github.com/stacksgov/sips/blob/feat/sip-021-nakamoto/sips/sip-021/sip-021-nakamoto.md). Therefore, this SIP is only meaningful when SIP-021 activates. The sBTC Working Group plans to observe at least 2-4 weeks of network behavior on Stacks Nakamoto to ensure a stable release. After this period, sBTC can be activated on the Stacks network without requiring a separate hard fork.
+
+### Process of Activation
+
+Users can vote to approve this SIP with either their locked/stacked STX or with unlocked/liquid STX, or both. The SIP voting page can be found at [sbtc.vote](https://sbtc.vote). The criteria for the stacker and non-stacker voting is as follows.
+
+#### For Stackers:
+
+In order for this SIP to activate, the following criteria must be met by the set of Stacked STX:
+
+- At least 80 million Stacked STX must vote, with least 66% (48 million) voting "yes".
```

Comment:
This SIP seems important enough to get high consensus of 80%. 
66% required for Stacked STX we have not done before.

So perhaps think about using below to match other SIPs.

* At least 80 million Stacked STX must vote at all to activate this SIP.
* Of the Stacked STX that vote, at least 80% of them must vote "yes."

And update non-Stackers vote power %  to 80% to match.

Reply:
@andrerserrano @aldur

Reply:
Updated to 70% to match the sBTC signer operations.